content/applications/finance/payment_acquirers/stripe.rst

```diff
@@ -14,22 +14,53 @@ Configuration
 Credentials tab
 ---------------
 
-Odoo needs your **API Credentials** to connect with your Stripe account, which comprises:
+The way you acquire your credentials depends on your hosting:
 
-- :ref:`Publishable Key <stripe/api_keys>`: The key solely used to identify the account with Stripe.
-- :ref:`Secret Key <stripe/api_keys>`: The key to sign the merchant account with Stripe.
-- :ref:`Webhook Signing Secret <stripe/webhook>`: When you enable your webhook on your Stripe
-  account, this signing secret must be set to authenticate the messages sent from Stripe to Odoo.
+- On Odoo Online, all you need to do is follow the onboarding.
```

Comment:
I'd just say "Odoo Online, follow the onboarding" but that might be too harsh @jcs-odoo?

Reply:
It hurts my feelings. I left "On Odoo Online, all you need to do is follow the onboarding." for now ;)

Reply:
@LoredanaLrpz @AntoineVDV 

On Odoo Online, please follow the onboarding.

Otherwise.
If your hosting plan appears to be with our SaaS solution, namely Odoo Online, please consider following the user-friendly onboarding.